packages/web-react/src/components/TextArea/TextArea.stories.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import TextArea from './TextArea';
+import { SpiritTextAreaProps } from '../../types';
+
+export default {
+  title: 'Components/TextArea',
+  argTypes: {
+    id: 'default',
+    placeholder: 'Placeholder',
+    isDisabled: {
+      control: 'boolean',
+    },
+    isRequired: {
+      control: 'boolean',
+    },
+    isLabelHidden: {
+      control: 'boolean',
+    },
+    isFluid: {
+      control: 'boolean',
+    },
+    maxLength: {
+      control: 'number',
+    },
+    rows: {
+      control: 'number',
+    },
+    validationState: {
+      control: {
+        type: 'select',
+        options: ['success', 'warning', 'error'],
+      },
+    },
+    label: 'Label',
+    message: 'Message',
+  },
+};
+
+export const Template = (args: SpiritTextAreaProps) => <TextArea {...args} />;
+
+export const Default = Template.bind({});
```

Comment:
This was copied from `TextField/`. Since I don't have that much insight into the Storybook, I'll follow how you – Javascript Ninjas – set it up here. 🙂 

I will edit it in the other form components when we are already cleaning up in the styles of form components #536.